### ISSUE TITLE
occ maintenance:install: The "--database-table-prefix" option does not exist.  

### ISSUE THREAD

**Issue by nodiscc** [nextcloud/server#24836]
## How to use GitHub

* Please use the 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to show that you are affected by the same issue.
* Please don't comment if you have no relevant information to add. It's just extra noise for everyone subscribed to this issue.
* Subscribe to receive notifications on status change and new comments. 

### Steps to reproduce
1. Install Nextcloud using the `occ` command documented at https://docs.nextcloud.com/server/latest/admin_manual/configuration_server/occ_command.html

### Expected behaviour

Running `occ maintenance:install` with documented options should result in a working Nextcloud installation.

### Actual behaviour

The command exits with this error:

```
  The "--database-table-prefix" option does not exist.  
```

In my specific case the command runs from a configuration management system (ansible). The full output of the task is as follows:

```
TASK [nextcloud : run first setup (this can take a while)] *********************************************************************************************************************************************************************************************************************
fatal: [REDACTED.lan]: FAILED! => {
    "changed": false,
    "cmd": [
        "/usr/bin/php",
        "occ",
        "maintenance:install",
        "--database=pgsql",
        "--database-host=localhost",
        "--database-name=nextcloud",
        "--database-table-prefix=oc_",
        "--database-user=nextcloud",
        "--database-pass=REDACTED",
        "--admin-user=REDACTED",
        "--admin-pass=REDACTED",
        "--data-dir=/var/nextcloud/data",
        "--admin-email=REDACTED"
    ],
    "delta": "0:00:00.416104",
    "end": "2020-12-23 19:55:18.599567",
    "failed_when_result": true,
    "rc": 1,
    "start": "2020-12-23 19:55:18.183463"
}

STDERR:


                                                        
  The "--database-table-prefix" option does not exist.  
                                                        

maintenance:install [--database DATABASE] [--database-name DATABASE-NAME] [--database-host DATABASE-HOST] [--database-port DATABASE-PORT] [--database-user DATABASE-USER] [--database-pass [DATABASE-PASS]] [--database-table-space [DATABASE-TABLE-SPACE]] [--admin-user ADMIN-USER] [--admin-pass ADMIN-PASS] [--admin-email [ADMIN-EMAIL]] [--data-dir DATA-DIR]


MSG:

non-zero return code
```

### Server configuration

**Operating system:** Debian GNU/Linux 10 (Buster)

**Web server:** Apache 2.4

**Database:** PostgreSQL

**PHP version:** 7.4

**Nextcloud version:** 20.0.2

**Updated from an older Nextcloud/ownCloud or fresh install:** Fresh install

**Where did you install Nextcloud from:** .zip from https://nextcloud.com/install/#instructions-server

**Signing status:** Not relevant

**List of activated apps:** None (incomplete setup)

**Nextcloud configuration:** Not relevant (incomplete setup)

**Are you using external storage, if yes which one:** No

**Are you using encryption:** no

**Are you using an external user-backend, if yes which one:** No

### Client configuration

Not relevant (setup fails on the command line/SSH)

#### Web server error log

Not relevant

#### Nextcloud log (data/nextcloud.log)

Not relevant

#### Browser log

None (no access from web browser)

-----------------------------------

It seems the `occ maintenance:install` command syntax was changed recently. https://docs.nextcloud.com/server/latest/admin_manual/configuration_server/occ_command.html still references `--database-table-prefix` as a valid option. I think the documentation should be updated to reflect that (is this related to `--database-table-space` ?).

Moreover I recall the `--database-table-prefix` to be a valid option on an earlier 20.x version. It seems that's no longer the case on 20.0.2. I do not expect a minor/patch release to break the `occ` command-line interface/syntax, so please consider that for future releases.

Thanks a lot

**Comment by kesselb** [comment]
Thanks for reporting :+1: 

> still references --database-table-prefix as a valid option. I think the documentation should be updated to reflect that

https://github.com/nextcloud/documentation/pull/5852

> Moreover I recall the --database-table-prefix to be a valid option on an earlier 20.x version. It seems that's no longer the case on 20.0.2. I do not expect a minor/patch release to break the occ command-line interface/syntax, so please consider that for future releases.

`--database-table-prefix` is available for **< Nextcloud 20.0.0**: https://github.com/nextcloud/server/pull/21112

**Comment by nodiscc** [comment]
Thanks for raising https://github.com/nextcloud/documentation/pull/5852, indeed `--database-table-prefix` was removed fro nextcloud 20.0.0 (should have read the release notes more closely) and the solution to my problem was quite simple - just remove `--database-table-prefix` from the install command.
